core/src/test/scala/integration/kafka/api/BaseQuotaTest.scala

```diff
@@ -235,14 +240,29 @@ abstract class QuotaTestClients(topic: String,
     numConsumed
   }
 
-  def verifyProduceThrottle(expectThrottle: Boolean, verifyClientMetric: Boolean = true): Unit = {
+  def verifyThrottleTimeRequestChannelMetric(apiKey: ApiKeys, metricNameSuffix: String,
```

Comment:
Could this be private?

Reply:
Yes. I also restricted access to a few other methods in this class where possible.